### ISSUE TITLE
Lower case and 'undefined' subfolder

### ISSUE THREAD

**Issue by airallergy** [wileyyugioh/zotmoov#50]
First of all, many thanks for making this plugin to make the transition to Zotero 7 much easier!

ZotFile allows to make all subfolder names in lower case, and will have a fallback subfolder called 'undefined' when the wildcard does not work, so that the files won't mixed with folders together.

I didn't seem to find relevant settings in zotmoov for these two functions, are they currently available? If not, would you consider adding them?

**Comment by wileyyugioh** [comment]
Yes, I think it's possible. I think I'll do it by adding a wildcard that just specifies "undefined." That way you can use the conditionals and still make it so that the fallback behavior for undefined wildcards is still ignore.

**Comment by wileyyugioh** [comment]
Please try prerelease [1.1.15](https://github.com/wileyyugioh/zotmoov/releases/tag/1.1.15). You can also change what the "undefined" string is by modifying the hidden preference ```extensions.zotmoov.undefined_str```

**Comment by airallergy** [comment]
`%U` works like a charm, thank you!

I didn't find a way to make the folder names all lower case, is it not in 1.1.15 yet?

**Comment by wileyyugioh** [comment]
Yeah it’s not in 1.1.15 yet. For 1.1.16 I’m planning to work on implementing something similar to the custom wildcards of Zotfile so you can do the toLowerCase and regex substitutions

**Comment by wileyyugioh** [comment]
You can try the [latest beta version's](https://github.com/wileyyugioh/zotmoov/releases/tag/1.2.0) custom wildcards. You can use the ```toLowerCase``` command.

**Comment by airallergy** [comment]
Seems to work perfectly, thanks!

It may be useful to include some examples in the documentation to make it clear how to customise wildcards. I wasn't sure whether `{}` is needed and had to try out.
